style.css

```diff
@@ -122,13 +114,430 @@ body {
     padding-left: 10%;
     padding-top: 40%;
     padding-right: 5%;
+    background-image: url("images/illustrations.png");
+    background-repeat: no-repeat;
+    background-origin: content-box;
+    background-position: bottom right;
+    background-position-x: 110%;
+    background-position-y: 120%;
   }
 
   .btn {
     border: none;
     outline: none;
     background-color: #1c1a19;
   }
+
+  .projects {
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+  }
```

Comment:
- You have done well so far, but according to the project requirements, you are to align the content of this section with the `grid` display property. Please fix this 😄